swift/Samples/TicTacToe/Sources/Game/RunGameWorkflow.swift

```diff
@@ -148,22 +150,23 @@ extension RunGameWorkflow {
                 ))
             ))
             
-            let confirmQuitScreen = ConfirmQuitWorkflow()
+            let (confirmQuitScreen, confirmQuitAlert) = ConfirmQuitWorkflow()
                 .mapOutput( { output -> Action in
                     switch output {
                     case .cancel:
                         return .startGame
-                    case .confirm:
+                    case .quit:
                         return .back
                     }
                 })
                 .rendered(with: context)
+                alert = confirmQuitAlert
             modals.append(ModalContainerScreenModal(screen: AnyScreen(confirmQuitScreen), style: .fullScreen, key: "0", animated: true))
         }
         
         let modalContainerScreen = ModalContainerScreen(baseScreen: BackStackScreen(items: backStackItems), modals: modals)
 
-        return modalContainerScreen // this is the base screen. Render all of the pieces of the backstack.
+        return AlertContainerScreen(baseScreen:modalContainerScreen, alert: alert) // this is the base screen. Render all of the pieces of the backstack.
```

Comment:
Nit: Not sure the comment is necessary

Reply:
Sorry, that comment was a reminder for myself (like the code version of the movie Memento). I will remove this.